tdx-attestation-app/ATTEST.md

```diff
@@ -0,0 +1,143 @@
+
+## **Performing Remote Attestation**
+
+* [Microsoft Azure in-guest app](#azure-in-guestapp)
+* [Intel Trust Authority in-guest app](#ita-in-guestapp)
+## <a id='azure-in-guestapp'></a> Azure in-guest app
+### Before you start: 
+1. You have successfully deployed your VM in Azure with Ubuntu.
+2. Install and build sample application
+-------------------------------------------------
+
+### Install and Build the Attestation Application
+
+1. Clone the Repository
+``` bash
+git clone https://github.com/Azure/confidential-computing-cvm-guest-attestation.git
+```
+
+2. Navigate to directory
+```
+cd confidential-computing-cvm-guest-attestation
+```
+
+3. Check preview branch
+``` bash
+git checkout tdx-preview
+```
+
+4. Navigate to Application Folder
+```
+cd tdx-attestation-app
+```
+
+5. Install Dependencies
+- Ubuntu
+```bash
+sudo ./install.sh
+```
+
+6. Configure and Build
+```sh
+cmake .
+make
+```
+7. To attest with MAA run the application with the following command:
+```sh
+sudo ./TdxAttest -c maa_config.json
+```
+9. To attest with Intel Trust Authority sign up to retrieve an API Key for [Intel Trust Authority](https://github.com/Azure/azure-compute-tdx-preview/blob/main/ITA.md)
```

Comment:
To attest with Intel Trust Authority, sign up to retrieve an API Key for [Intel Trust Authority] (https://www.intel.com/content/www/us/en/security/trust-authority.html). For more information on Intel Trust Authority see their [documentation](https://docs.trustauthority.intel.com/main/articles/introduction.html).